tests/container_hardening/test_container_hardening.py

```diff
@@ -10,16 +10,20 @@
 
 NO_PRIVILEGED_CONTAINERS = [
     'bgp',
+    'lldp',
+    'teamd'
 ]
 
 
-def test_container_privileged(duthost):
+def test_container_privileged(duthost, enum_asic_index):
```

Comment:
>test_container_privileged [](http://example.com/codeflow?start=4&length=25)

There are some containers having single instance in multi-asic, for example snmp/telemetry. Is this function flexible enough to cover all the cases in future?

Reply:
Yes, lldp, included in this PR, is also a single instance container in multi-asic

Reply:
Even if the container is a single instance container, the test case will be executed for all asics.
So for 3 asic duthost, test will be called with enum_asic_index 0, then 1 and then 2.
Should we execute this test only for any randomly selected asic? assuming that if lldp0 privilege flag is verified lldp1 will have the same privilege level ? If yes, then we could use "enum_rand_one_asic_index" instead on enum_asic_index

Reply:
Yes, lldp1 will have same privilege level if lldp0 is verified, changed to `enum_rand_one_asic_index`